django/test/runner.py

```diff
@@ -5,30 +5,226 @@
 import os
 import pickle
 import textwrap
+import time
 import unittest
+import warnings
 from importlib import import_module
 from io import StringIO
 
 from django.core.management import call_command
+from django.core.management.color import color_style, no_style, supports_color
 from django.db import connections
 from django.test import SimpleTestCase, TestCase
 from django.test.utils import (
     setup_databases as _setup_databases, setup_test_environment,
     teardown_databases as _teardown_databases, teardown_test_environment,
 )
 from django.utils.datastructures import OrderedSet
+from django.utils.termcolors import RESET
 
 try:
     import tblib.pickling_support
 except ImportError:
     tblib = None
 
 
-class DebugSQLTextTestResult(unittest.TextTestResult):
-    def __init__(self, stream, descriptions, verbosity):
+TERMINATOR = '\x1b[{}m'.format(RESET)
+
+
+def get_traceback_highlighter(bg='light'):
```

Comment:
This is only ever called once. Do we need the default? 
(Same with SQL version)